conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However it's important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
```

Comment:
it looks like students and team members are used interchangeably. If there is a difference between the two, there should be a sentence clarifying the distinction.

Reply:
I mentioned team members to drive in that it's expected of all students and also in the group setting, therefor, as long as it isn't confusing, I'm going to keep this line how it is.